addons/website/static/src/snippets/s_popup/000.js

```diff
@@ -71,6 +71,7 @@ const PopupWidget = publicWidget.Widget.extend({
         if (this._popupAlreadyShown) {
             return;
         }
+
```

Comment:
remove this empty line

Reply:
If you check overall diff, there is no extra line added.

Reply:
ok so dont add a line in your first commit to remove it in the second one ;)